Dockerfile

```diff
@@ -1,15 +1,23 @@
 FROM php:7.1-apache
 
+ARG USE_C_PROTOBUF=false
```

Comment:
Why is this false by default? I thought we'd switch to using the C extension by default.

Reply:
Good point. I should have detailed that the purpose of this PR is to add the build argument only: I set the default as false to not introduce any breaking changes.

The change to Protobuf C will be done by other PR(s) and changing the value to true will be part of it.

Reply:
Got it, thanks.